pandunia-loge.csv

```diff
@@ -2308,100 +2308,100 @@ Luter din ja||||Lutheran|luthérien|luterano|luterano|лютеранский||||
 Makau|desha|MO||Macao||Macao||||||||||||||Macao|Makau
 mal|||ara:fas:urd: مَال‎ (māl), hin:माल (māl), tur:mal, swa:mali, may:milik|property (belongings, possessions)|propriété (possession, bien)|propiedad|||مَال‎|||||माल (संपत्ति)||||||omaisuus|własność (mienie)
-mala||||own (possess, have)||poseer (tener, ser dueño de)||||||||||||||omistaa|mieć (posiadać)
-malagasi||||Malagasy||malgache||||||||||||||malagassi|madagaskarski; malagaski
-Malagasia|desha|MG||Madagascar||Madagascar||||||||||||||Madagaskar|Madagaskar
+fa mal||||own (possess, have)||poseer (tener, ser dueño de)||||||||||||||omistaa|mieć (posiadać)
+madagaskar di||||Malagasy||malgache||||||||||||||malagassi|madagaskarski; malagaski
+Madagaskar|desha|MG||Madagascar||Madagascar||||||||||||||Madagaskar|Madagaskar
 Malaisia|desha|MY||Malaysia||Malasia||||||||||||||Malesia|Malezja
-malan||||owner (proprietor)|possédant (propriétaire)|dueño|||||||||||||havanto|omistaja (pitäjä, hallussapitäjä)|właściciel (posiadacz)
+mal ja||||owner (proprietor)|possédant (propriétaire)|dueño|||||||||||||havanto|omistaja (pitäjä, hallussapitäjä)|właściciel (posiadacz)
 malaria|||eng:spa:fra:swa:may:malaria, por:malária, rus:малярия (mal′arija), tur:malarya, ara:مَلَارِيَا‎ (malāriyā), hin:मलेरिया (maleriyā), jpn:マラリア (mararia)|malaria||malaria (paludismo)||||||||||||||malaria|malaria
 Malavi|desha|MW||Malawi||Malaui||||||||||||||Malawi|Malawi
 malayali|nas|||Malayali (Malayalam)||malayali (malabar, malayalam)||||||||||||||malajalam (eräs intialainen kieli)|malajski
-malayi||||Malay||malayo||||||||||||||malaiji|malajski
+malayu||||Malay||malayo||||||||||||||malaiji|malajski
 Mali|desha|ML||Mali||Mali|||||||||||||Malio|Mali|Mali
-malnomia||||economics|économie|economía|economia|экономика|اِقْتِصَاد‎|经济学|経済学|경제학|kinh tế học|अर्थशास्त्र (अर्थनीति)|অর্থনীতি|ekonomi|uchumi|iktisat|ekonomiko|taloustiede (ekonomia)|ekonomia
+mal nomi||||economics|économie|economía|economia|экономика|اِقْتِصَاد‎|经济学|経済学|경제학|kinh tế học|अर्थशास्त्र (अर्थनीति)|অর্থনীতি|ekonomi|uchumi|iktisat|ekonomiko|taloustiede (ekonomia)|ekonomia
 Malta|desha|MT||Malta||Malta||||||||||||||Malta|Malta
-malu||||belong|appartenir à|ser de||принадлежать||属于|||||||||aparteni|kuulua (olla jkn hallussa)|należeć
+be mal||||belong|appartenir à|ser de||принадлежать||属于|||||||||aparteni|kuulua (olla jkn hallussa)|należeć
 Malvinas nesia|desha|FK||Falkland Islands (Malvinas)||Islas Malvinas||||||||||||||Falklandin saaret|Falklandy, Malwiny
-mam|||eng:mamma, fra:mamelle, spa:por:mama, tur:meme|breast (boob, tit)|sein|seno (teta)|seio (mama)|||||||||payudara|titi|meme|mamo|nisä (tissi)|
+titi|||eng:tit, fra:tété, spa:teta, rus:титьки (tit’ki), ara:ثَدْي‎ (ṯady), swa:titi, may:tetek, jpn:乳 (chichi)|breast (boob, tit)|sein|seno (teta)|seio (mama)|||||||||payudara|titi|meme|mamo|nisä (tissi)|
 mamal|bio|Mammalia|eng:mammal, spa:por:mamífero, fra:mammifère, tur:memeli, may:mamalia|mammal||mamífero||||||||||||||nisäkäs|ssak
```

Comment:
I preferred the connection between "mam" and "mamal" even though -al is not a real suffix in Pandunia.